src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_existingLogin_notOk() {
+        User user = new User();
+        user.setLogin("Good login");
+        user.setPassword("Good password");
+        user.setAge(19);
+
+        registrationService.register(user);
+
+        User newUser = new User();
+        newUser.setLogin("Good login");
+        newUser.setPassword("Good password");
+        newUser.setAge(19);
+
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(newUser),
+                "Expected register() to throw, but it didn't");

```

Comment:
The name of the test method `register_existingLogin_notOk` does not reflect the edge cases. Consider including the condition that triggers the exception in the method name, like `register_duplicateLogin_shouldThrowException`.